tests/test_pivot.py

```diff
@@ -239,6 +276,27 @@ def test_pivot_longer_names_sep_regex(self):
         self.assertTrue(result)
 
 
+    def test_pivot_longer_names_sep_position(self):
+        """ Test if pivot_longer works with names_sep argument being a position
+        """
+        stang = data.df_stang_wide
+        long = gr.tran_pivot_longer(
+            stang,
+            names_sep=-3,
```

Comment:
The `sep` argument in `tran_separate()` is provided as a list; this is so one can specify multiple separation points for multiple `into` entries. The `names_sep` argument should provide similar functionality with `names_to`; i.e. one should be able to provide arguments where `len(names_to) - 1 == len(names_sep)`.